docs/tutorial.md

```diff
@@ -344,6 +344,45 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace.
 
+## SRE Recipes
+
+SRE Recipes is our [Chaos Engineering](https://en.wikipedia.org/wiki/Chaos_engineering) tool to test your sandbox environment. It helps users to familiarize with finding the root cause of a breakage using Cloud Ops monitoring.  
+Each 'recipe' simulate a different scenario, there are several recipes that you can run and you can also [contribute your own.](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/sre-recipes#contributing)  
+
+```
+$ sandboxctl sre-recipes  
+```
+
+### Running an example SRE Recipe
+
+> **Note:** Recipe's names are not explicit by design as we don't want allude to the problem.
+
+1. Run the recipe to manufacture errors in the demo cluster
+
+```
+$ sandboxctl sre-recipes break recipe0
+```
+
+2. Use Cloud Operations suite to diagnose the problem.
+
+> **Note:** If you are stuck, you can use a hint to direct you to the right direction.
+```
+$ sandboxctl sre-recipes hint recipe0
+```
+
+3. Verify you hypothesis using command line tool
```

Comment:
Verify your hypothesis on what could be wrong with the demo app by using command line tool